### PR TITLE
PP-8986 Read internal events from queue

### PR DESCRIPTION
**Pull request by alexbishop1**
Read internal events (from elsewhere in the platform) from the SQS queue and process them to send webhook messages.

### CODE REVIEW

File: src/main/java/uk/gov/pay/webhooks/queue/QueueMessageReceiver.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                .threads(queueReadScheduleNumberOfThreads)
+                .build();
+    }
+
+    @Override
+    public void start() {
+        long initialDelay = config.getThreadDelayInMilliseconds();
+        long delay = config.getThreadDelayInMilliseconds();
+
+        for(int i = 0; i < queueReadScheduleNumberOfThreads; i++) {
```
Comment (gidsg) [inline src/main/java/uk/gov/pay/webhooks/queue/QueueMessageReceiver.java:46]: could we do this as a forEach instead?

**Review by gidsg — APPROVED** [review]
Looks good to me, aside from minor comment 👍 👑

File: src/main/java/uk/gov/pay/webhooks/app/SqsConfig.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    private String accessKey;
+    private String secretKey;
+
+    @Max(20)
+    private int messageMaximumWaitTimeInSeconds;
+    @Max(10)
+    private int messageMaximumBatchSize;
+
+
+    private boolean nonStandardServiceEndpoint;
```
Comment (sfount) [inline src/main/java/uk/gov/pay/webhooks/app/SqsConfig.java:24]: I think using language like "AWS environment" or "local environment" might help future us understand this without as much README. No suggestions at this hour, happy for it to be in a follow up after discussion

File: src/main/java/uk/gov/pay/webhooks/queue/QueueMessageReceiver.java (hunk 1 above)
Comment (alexbishop1) [reply src/main/java/uk/gov/pay/webhooks/queue/QueueMessageReceiver.java:46]: Not really. `queueReadScheduleNumberOfThreads` is just an `int`. There’s no collection we’re iterating over here.

File: src/main/java/uk/gov/pay/webhooks/app/SqsConfig.java (hunk 2 above)
Comment (alexbishop1) [reply src/main/java/uk/gov/pay/webhooks/app/SqsConfig.java:24]: Yeah, let’s change this in all the queue-consuming projects at once.

**Pull request merged**